ui/imports/shared/views/chat/ProfileContextMenu.qml

```diff
@@ -38,8 +38,7 @@ StatusMenu {
     signal removeFromGroup
 
     ProfileHeader {
-        width: parent.width
-        height: visible ? implicitHeight : 0
+        implicitWidth: parent.width
```

Comment:
hmm, `implicitWidth` shouldn't be set externally. Why not just `width`?

Reply:
It doesn't work, tried. It just doesn't set the correct width, most likely because we are inserting this static (non MenuItem) entry into the PopupMenu

Reply:
This is with `width`:
![image](https://github.com/user-attachments/assets/65573f04-10b3-4e85-bec6-5d362a9be4e2)

Reply:
Ah, probably because it goes to `ColumnLayout` finally:

![image](https://github.com/user-attachments/assets/ee7b11e7-66a7-489b-9b84-97490afc9129)

Then interesting why `item.Layout.fillWidth = true` is not doing the job.

Reply:
Tried that too; same result :)

Reply:
Maybe because it isn't added to the `contentModel`

Reply:
Something is substantially wrong with this sizing, but I didn't find what after my short investigation...

Interesting fact:

Setting only `id`, like `id: col` in `StatusMenu` for `ColumnLayout` (line 97) results with:
![image](https://github.com/user-attachments/assets/0b2c3c10-5b3f-4c09-ae57-077f79db66dd)

:thinking:

Reply:
```qml
ColumnLayout {
    Repeater {
        model: root.contentModel

        onItemAdded: {
            item.Layout.fillWidth = true
            item.Layout.maximumWidth = root.maxImplicitWidth
        }
    }
}
```
the width of this column layout is unbound, so the fillWidth means expanding to the width of the widest element, also making items centered (because text inside is aligned to center)

So when there is only header, natural implicit width of that component is just smaller than minimal size of the menu (normally we have other entries causing expansion). So when you artificially bumps the implicit width, it fits well.

So the proper solution is to set minimal width on added items, not less then menu width:
```qml
contentItem: StatusScrollView {
    id: scrollView

    padding: 0

    ColumnLayout {
        Repeater {
            model: root.contentModel

            onItemAdded: {
                item.Layout.fillWidth = true
                item.Layout.minimumWidth = scrollView.width
                item.Layout.maximumWidth = root.maxImplicitWidth
            }
        }
    }
}
```
And then there is no need to set `implicitWidth` on `ProfileHeader` as a workaround

Reply:
Fixed, thanks!